toolchain/check/testdata/class/virtual_modifiers.carbon

```diff
@@ -164,6 +164,81 @@ class Derived {
   impl fn F();
 }
 
+// --- fail_impl_mismatch.carbon
+
+package FailImplMismatch;
```

Comment:
I'm noticing, why do you do this instead of `library `[[@TEST_NAME]]";`?

Reply:
ah, thanks! did that in c72a43acf9a881e40cb5e7e3da3a55918e5525c1